src/admin-events/admin-events.spec.js

```diff
@@ -0,0 +1,106 @@
+const ethers = require('ethers');
+const { getAddress } = require('@uma/contracts-node');
+
+// Load Config files
```

Comment:
this "Load Config Files" comment doesn't match the code

Reply:
Done